src/lib/config/defaultConfig.js

```diff
@@ -318,7 +318,20 @@ export const RECORDS_CONFIG = {
     },
   },
   DOCUMENT_REQUESTS: {
-    states: ['ACCEPTED', 'PENDING', 'DECLINED'],
+    states: [
+      {
+        text: 'ACCEPTED',
+        default: false,
```

Comment:
Very minor: I would keep the `default` only where it is `true` and, when used, check that if the `default` property does not exist, it will be `false`. I saw this pattern introduced recently in other places, might be a good idea to fix all the `default: false` occurrences and their usage.

Reply:
fair !